spec/granite_orm_spec.cr

```diff
@@ -12,4 +13,18 @@ describe Granite::ORM::Base do
     t = Todo.new(name: "Elorest")
     t.name.should eq "Elorest"
   end
+
+  it "should provide a to_h method" do
+    t = Todo.new(name: "test todo", priority: 20)
+    result = { "name" => "test todo", "priority": 20 }
```

Comment:
This is interesting:

```crystal
{ "name" => "test todo", "priority": 20 }.class # => Hash(String, Int32 | String)
{ "priority": 20 }.class # => NamedTuple(priority: Int32)
{ "priority": 20, "name" => "test todo" }.class # space not allowed between named argument name and ':'
```

I don't want to be pedantic, but the first case looks like a bug in Crystal or at least it is not documented. I would prefer to use here official way for hashes to avoid problems in future:

```crystal
{ "name" => "test todo", "priority" => 20 }.class # => Hash(String, Int32 | String)
```

Reply:
@veelenga My ruby background seems to have seeped in here pushing a fix shortly.